src/vsr/grid_scrubber.zig

```diff
@@ -127,6 +127,9 @@ pub fn GridScrubberType(comptime Forest: type) type {
         /// Contains a table index block when tour=table_data.
         tour_index_block: BlockPtr,
 
+        /// These counters reset after every tour cycle.
+        tour_blocks_scrubbed_count: usize,
```

Comment:
micronit, but let's use `u64`. Number of blocks scrubbed is a property independent of host architecture